ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java

```diff
@@ -134,13 +135,13 @@ public boolean isSuccessful() {
   protected AbstractBlockProcessor(
       final MainnetTransactionProcessor transactionProcessor,
       final TransactionReceiptFactory transactionReceiptFactory,
-      final Wei blockReward,
+      final Function<Long, Wei> blockRewardCalculator,
```

Comment:
hoping we don't need a calculator - but rather, we can ensure the blockProcessor at each Milestone knows what reward to expect.

Reply:
Yeah... just passed scheduler to block processor